src/services/serverApi.ts

```diff
@@ -13,18 +13,18 @@ export const serverApi = createApi({
       return action.payload[reducerPath];
     }
   },
-  tagTypes: ['Skill', 'Contributions', 'ActiveTasks', 'Members', 'User'],
+  tagTypes: ['Skill', 'Contributions', 'ActiveTasks', 'AllUsers', 'User'],
   endpoints: (builder) => ({
     // Queries
-    getMembers: builder.query<MembersResponseType, void>({
-      query: () => BASE_URL + '/members',
-      providesTags: ['Members']
+    getAllUsers: builder.query<UsersResponseType, void>({
+      query: () => BASE_URL + '/users',
+      providesTags: ['AllUsers']
```

Comment:
We are replacing `/members` with `/users` API
But users API is paginated, and members API is not paginated 
Won't it break when the results are being paginated from users API?
Or am I missing something here?

Reply:
Initially the paginated version of `/users` API is not being implemented and this will be addressed in a dedicated issue https://github.com/Real-Dev-Squad/members-site/issues/101 which was concluded after a discussion with vinayak.

Reply:
@kotesh-arya as discussed you still need to pass the size in the query right?

Reply:
Yeah, I have added the query `size=100` to the /users API as per our discussion.